system_tests/program_test.go

```diff
@@ -423,9 +424,12 @@ func storageTest(t *testing.T, jit bool) {
 
 	validateBlocks(t, 2, jit, builder)
 
-	// Captures a block_input_<id>.json file for the block that included the
-	// storage write transaction.
-	recordBlock(t, receipt.BlockNumber.Uint64(), builder)
+	// Captures a block_inputs json file for the block that included the
+	// storage write transaction. Include wasm targets necessary for arbitrator prover and jit binaries
+	flag.Parse()
```

Comment:
I *think* you can now move the flag parsing and the checking of "recordBlockInputsEnable" into the implementation of recordBlock.

Reply:
I agree